### ISSUE TITLE
Add Popup Title or custom field as CSS identifier

### ISSUE THREAD

**Issue by dbmpls** [PopupMaker/Popup-Maker#669]
As discussed with Daniel - 

There were some plans to remove the title from the popup css classes, but I believe this is an important feature - especially for multisite users.

Adding the popup title as a css class allows users to target popups or groups of pop-ups more easily on a multisite network. 

An alternative that would be more persistent as pop-ups are updated across a multisite network would be to add a field for CSS Class on each pop-ups advanced or display settings.

**Comment by danieliser** [comment]
@dbmpls - I remember what we were discussing, but not sure how this plays in. Popups already get click trigger classes of popmake-name-slug. 

Are asking that the Popup div itself gets a class of pum-name-slug?

**Comment by dbmpls** [comment]
Apologies if I misunderstood part of our conversation.  In regards to targeting pop-ups for custom css and jquery, I thought you mentioned that popup slugs and popup theme names were going to be removed from popups html and that only  IDs would be present.

I had asked that you keep them in place, and you said to add it to github, but maybe we were thinking about different things.

For specific examples of code that I am using:  I currently am using the following where the pop-up slug is "site-tour"

`<script type="text/javascript">
jQuery('[data-popmake*="site-tour"]')
    .on('pumBeforeOpen', function () {
		gaplusu('set', 'page', '/site-tour-popup');
		gaplusu('send', 'pageview');
    });
</script>`

and 

`div[data-popmake*="site-tour"]{

	...mycustom css
}`

As long as the popup slug remains in data-popmake, or instead was added as a css class directly to the div, users like myself would be ok.

Bottom line - I just need a continued way going forward to target pop-ups that doesn't involve popup post-id. 

If it was never a plan to remove that data, chalk it up to a misunderstanding and feel free to close.

Thanks again!

**Comment by danieliser** [comment]
@dbmpls I believe I was talking about the Popup Title in the admin popup editor. That field is being removed when using the Gutenberg editor. Instead you will be able to either add the `pum-title` class to any block, or there will be a new Popup Title block when editing popups.

But in general gonna close this one.
